### PR TITLE
Tickets/sitcom 1223

### PR DESCRIPTION
**Pull request by ltoribiosc**

### CODE REVIEW

**Review by bstalder — CHANGES_REQUESTED** [review]
please remove the 1136 file changes.  also clear outputs to the 1223 notebook.  also inline plots are preferred instead of file outputs.

**Comment by ltoribiosc** [comment]
Thanks Brian. I just deleted the notebook outputs and deleted the 1136 files.

**Comment by bstalder** [comment]
think you need further commits.  I still see the M1M3 notebook with changes.

**Comment by ltoribiosc** [comment]
Where do you see the M1M3 notebook changes? I deleted all the files that were copied from the previous ticket. You should see nothing now. 

I now see 13 commits all related to SITCOM-1223. That is, from this ticket

**Comment by bstalder** [comment]
I still see it under "file changes"

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb
Comment (ltoribiosc) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb:1]: This file was introduced due to an error when creating the branches. The SITCOM-1223 branch was not created from develop branch but from SITCOM-1136 by mistake. Therefore, in order to merge them it was necessary to copy the last files of SITCOM-1136 here

**Review by ltoribiosc — COMMENTED** [review]
SITCOM-1136 changes have been reviewed. This file was introduced due to an error when creating the branches. The SITCOM-1223 branch was not created from develop branch but from SITCOM-1136 by mistake. Therefore, in order to merge them it was necessary to copy the last files of SITCOM-1136 here

**Comment by ltoribiosc** [comment]
<p>Hi Brian. I didn't understand what you were saying. But I think it's fixed now. Tell me if this is the case or if I should do something else. Thank you very much for your help.</p>

**Review by bstalder — APPROVED** [review]

**Pull request merged**